persist-cli/src/main/java/io/ballerina/persist/nodegenerator/SyntaxTreeConstants.java

```diff
@@ -29,35 +31,52 @@
 public class SyntaxTreeConstants {
 
     private SyntaxTreeConstants() {}
-
-    public static final Token SYNTAX_TREE_SEMICOLON = AbstractNodeFactory.createIdentifierToken(";");
-    public static final Token SYNTAX_TREE_COLON = AbstractNodeFactory.createIdentifierToken(":");
-    public static final Token SYNTAX_TREE_OPEN_BRACE = AbstractNodeFactory.createIdentifierToken("{");
-    public static final Token SYNTAX_TREE_CLOSE_BRACE = AbstractNodeFactory.createIdentifierToken("}");
-    public static final Token SYNTAX_TREE_OPEN_PAREN = AbstractNodeFactory.createIdentifierToken("(");
-    public static final Token SYNTAX_TREE_CLOSE_PAREN = AbstractNodeFactory.createIdentifierToken(")");
-    public static final Token SYNTAX_TREE_OPEN_BRACKET = AbstractNodeFactory.createIdentifierToken("[");
-    public static final Token SYNTAX_TREE_CLOSE_BRACKET = AbstractNodeFactory.createIdentifierToken("]");
-    public static final Token SYNTAX_TREE_EQUAL = AbstractNodeFactory.createIdentifierToken("=");
-    public static final Token SYNTAX_TREE_PIPE = AbstractNodeFactory.createIdentifierToken("|");
-    public static final Token SYNTAX_TREE_SLASH = AbstractNodeFactory.createIdentifierToken("/");
-    public static final Token SYNTAX_TREE_COMMA = AbstractNodeFactory.createIdentifierToken(",");
+    public static final Token SYNTAX_TREE_SEMICOLON = AbstractNodeFactory.createToken(SyntaxKind.SEMICOLON_TOKEN);
+    public static final Token SYNTAX_TREE_COLON = AbstractNodeFactory.createToken(SyntaxKind.COLON_TOKEN);
+    public static final Token SYNTAX_TREE_OPEN_BRACE = AbstractNodeFactory.createToken(SyntaxKind.OPEN_BRACE_TOKEN);
+    public static final Token SYNTAX_TREE_CLOSE_BRACE = AbstractNodeFactory.createToken(SyntaxKind.CLOSE_BRACE_TOKEN);
+    public static final Token SYNTAX_TREE_OPEN_PAREN = AbstractNodeFactory.createToken(SyntaxKind.OPEN_PAREN_TOKEN);
+    public static final Token SYNTAX_TREE_CLOSE_PAREN = AbstractNodeFactory.createToken(SyntaxKind.CLOSE_PAREN_TOKEN);
+    public static final Token SYNTAX_TREE_OPEN_BRACKET = AbstractNodeFactory.createToken(SyntaxKind.OPEN_BRACKET_TOKEN);
+    public static final Token SYNTAX_TREE_CLOSE_BRACKET = AbstractNodeFactory.createToken(
+            SyntaxKind.CLOSE_BRACKET_TOKEN);
+    public static final Token SYNTAX_TREE_EQUAL = AbstractNodeFactory.createToken(SyntaxKind.EQUAL_TOKEN);
+    public static final Token SYNTAX_TREE_PIPE = AbstractNodeFactory.createToken(SyntaxKind.PIPE_TOKEN);
+    public static final Token SYNTAX_TREE_SLASH = AbstractNodeFactory.createToken(SyntaxKind.SLASH_TOKEN);
+    public static final Token SYNTAX_TREE_COMMA = AbstractNodeFactory.createToken(SyntaxKind.COMMA_TOKEN);
```

Comment:
Do we need separate constants for this? Cant we use this in the code itself? From the looks of it this does not look like that much complicated to need separate constants from persist module.

Reply:
As these are constants, Tokens were included here to increase the readability while keeping the number of lines in the method to minimum.